modules/openapi-generator/src/main/resources/C-libcurl/cmake-config.mustache

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
```

Comment:
If a project has its own "Config.cmake.in", your template file will replace it...